debian.oracle-5.15/config/annotations

```diff
@@ -84,3 +84,5 @@ CONFIG_LD_VERSION                               policy<{'amd64': '23400', 'arm64
 CONFIG_PAHOLE_VERSION                           policy<{'amd64': '121', 'arm64': '121'}>
 CONFIG_SLS                                      policy<{'amd64': '-'}>
 CONFIG_ZERO_CALL_USED_REGS                      policy<{'amd64': '-', 'arm64': '-'}>
+
+include "../../debian.delphix/config/delphix_annotations"
```

Comment:
doesn't really matter.. but since we already have the delphix specific directory `debian.delphix`.. the file name could just be `annotations` rather than `delphix_annotations`..

Reply:
done